src/search/FindReplace.js

```diff
@@ -612,6 +614,10 @@ define(function (require, exports, module) {
         // Prepopulate the search field
         var initialQuery = FindBar.getInitialQuery(findBar, editor);
         
+        if (initialQuery.query === "" && editor.lastParsedQuery !== "") {
+            initialQuery.query = editor.lastParsedQuery;
```

Comment:
This is not working well with regular expression. Further manual testing and a new test case are needed